controllers/enqueue.go

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright (c) 2021 Red Hat, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controllers
+
+import (
+	"github.com/redhat-appstudio/service-provider-integration-operator/pkg/logs"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var enqueueLog = log.Log.WithName("enqueue-logger")
+
+func logReconciliationRequests(requests []ctrl.Request, requestKind string, cause client.Object, objectKind string) {
```

Comment:
`enqueue.go` sounds quite important. What is responsibility of this file? Any plans to have anything else here than "just" logging?

Reply:
I was hoping to extract some of the logic that we have in our watchers into a separate file in line with our aim to split up the reconcilers. At the same time, I didn't want to dilute this PR with a refactoring.